resources/public/api/conf/1.0/errors.raml

```diff
@@ -188,7 +188,7 @@ traits:
             type: types.errorResponse
             examples:
               ruleTypeOfClaim:
-                description: The type of claim supplied is not valid.
+                description: The claim type selected is not available for this type of loss.
```

Comment:
This should be from the scenario column in the requirements spec. Correct before change, I think?